client/my-sites/pages/virtual-page/index.tsx

```diff
@@ -26,6 +27,7 @@ interface Props {
 	description?: string;
 	previewUrl?: string;
 	isHomepage?: boolean;
+	isAdmin: boolean;
```

Comment:
Super nit: I'd prefer this to be "grouped" with the `record*` props below, to indicate that it is part of the "connected" props :)

Reply:
Cool